src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -38,15 +38,24 @@ def user_icon(user, size = 20, css_class = nil, alt = nil)
               width: size, height: size, alt: alt, class: css_class)
   end
 
-  def fuzzy_time(time)
+  def fuzzy_time(time, opts = {})
+    output = nil
+
     if Time.now - time < 60
-      return 'now' # rails' 'less than a minute' is a bit long
+      output = 'now' # rails' 'less than a minute' is a bit long
```

Comment:
why not writing this like
`output = Time.now - time < 60 ? 'now' : time_ago_in_words(time) + ' ago'`